static/js/marker.js

```diff
@@ -43,7 +43,7 @@ var MarkerView = Backbone.View.extend({
 
         if (this.model.get("provider_code") == PROVIDER_CODE_UNITED_HATZALA) {
             this.marker.setIcon(this.getIcon());
-            this.marker.setTitle(this.getTitle('united'));
+            this.marker.setTitle(this.model.get("title"));
```

Comment:
The constructed title contains the time description, as far as I understand. Why include it for LMS accidents but not for United ones? (Also, see my other comment)